roxie/ccd/ccdfile.cpp

```diff
@@ -998,9 +998,9 @@ class CRoxieFileCache : implements IRoxieFileCache, implements ICopyFileProgress
                     if (todo.ordinality())
                     {
                         ILazyFileIO *popped = &todo.popGet();
-                        if (popped->isAlive())
+                        if (popped->isAliveAndLink())
```

Comment:
I think this could have caused problems - I vaguely remember crashes in this area.